src/Model/GatewayConfigInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommerceWeavers\SyliusTpayPlugin\Model;
+
+use Payum\Core\Model\GatewayConfigInterface as BaseGatewayConfigInterface;
+use Payum\Core\Security\CryptedInterface;
+
+interface GatewayConfigInterface extends BaseGatewayConfigInterface, CryptedInterface
```

Comment:
Is this needed? I can't find an implementation of this interface.

Reply:
We have problem of not having CryptedInterface on the object within Sylius. But I've managed with workaround